docs/source/md/quick-start/integrations.md

```diff
@@ -0,0 +1,74 @@
+## Integrations
+
+Easily integrate Aim with your favorite framework / tool
+
+### Python script
+
+```python
+import aim
+
+# Save inputs, hparams or any other `key: value` pairs
+aim.set_params(hyperparam_dict, name='hparams') # Passing name argument is optional
```

Comment:
Lets use the current API for this. The top-level functions, such as `set_params`, `track`, `select` are kept for the sake of beckrads compatibility. Not sure we want to even document them.

Reply:
dropping this comment as well. we have a task for API reference tidy-up.